packages/ra-ui-materialui/src/list/ListToolbar.tsx

```diff
@@ -63,18 +63,16 @@ export interface ListToolbarProps
 
 const PREFIX = 'RaListToolbar';
 
-const Root = styled(Toolbar, {
+const Root = styled('div', {
```

Comment:
The ListToolbar renders ListActions, which itself renders TopToolbar, which also renders a Toolbar. So there was a toolbar inside a toolbar, with added margin and padding... not really necessary.